rdtools/plotting.py

```diff
@@ -92,8 +98,14 @@ def degradation_summary_plots(yoy_rd, yoy_ci, yoy_info, normalized_yield,
     ax2.set_xlabel('Annual degradation (%)')
 
     renormalized_yield = normalized_yield / yoy_info['renormalizing_factor']
-    ax1.plot(renormalized_yield.index, renormalized_yield, 'o',
-             color=plot_color, alpha=scatter_alpha)
+    if detailed:
+        def f(x): return {0: 'red', 1: 'green', 2: plot_color}[x]
+        colors = yoy_info['usage_of_points'].apply(f)
```

Comment:
I think using a wrapper function with `apply` can be replaced with `yoy['usage_of_points'].map({0: 'red', 1: 'green', 2: plot_color})`.

Reply:
Nice!  I figured there had to be a simpler way, but couldn't figure it out.